src/utils/slack_notification.py

```diff
@@ -0,0 +1,111 @@
+import json
+import requests
+import re
+from tornado.httpclient import HTTPRequest, AsyncHTTPClient
+
+try:
+	from config import SLACK_WEBHOOKS
+except ImportError: 
+	SLACK_WEBHOOKS = []
+
+def get_tags(data):
+	"""Generate array of all the tags listed in the newly registered API"""
+	tags = []
+	if('tags' in data):
+		for item in data['tags']:
+			tags.append(item['name'])
+	return tags
+
+def change_link_markdown(description):
+	"""Change markdown styling of links to match fit Slack Markdown styling
+	
+	Description text links formatted as [link name](URL), we want <URL|link name>
+	"""
+	description = description.split(" ")
+	# check each word in string
+	for i in range(len(description)): 
+		if((description[i][0] == '[') and (description[i][-1] == ")") and ("](" in description[i])):
+			description[i] = description[i].replace("[","").replace(")","")
+			temp_list = description[i].split("](")
+			description[i] = "<" + temp_list[1] + "|" + temp_list[0] + ">"
+	return " ".join(str(x) for x in description)
```

Comment:
Alternatively, you may use this regex sub function:
```
return re.sub('\[(?P<label>[^\[\]]+)\]\((?P<url>[^()]+)\)', '<\g<label>|\g<url>>', description)
```

I tested with this example:
```
re.sub('\[(?P<label>[^\[\]]+)\]\((?P<url>[^()]+)\)', '<\g<label>|\g<url>>', 'aafadfasfasfaf [a](http://b.com) adsfaf [b](https://ccc.com) adadf')
```

more about `re.sub` here: https://docs.python.org/3/library/re.html#re.sub